### ISSUE TITLE
Unsucessful enrolment message spams "All" site admin users.

### ISSUE THREAD

**Issue by danmarsden** [ArloSoftware/moodle-enrol_arlo#129]
Some sites have a lot of Moodle admins - and all are notified when arlo runs into difficulties.

https://github.com/ArloSoftware/moodle-enrol_arlo/blob/4b6e15e5207ac4f377738d71aaaee929e2622711/classes/local/administrator_notification.php#L63

It would be preferable to have a way to either specify a contact person to receive these, or even just spam the main admin only - and not all admins in the site.

**Comment by troywilliams** [comment]
Yep, it can get pretty nasty! I don't know what was thinking at the time of writing that. It can be switched off in notification preferences. Thinking about moving to the daily email, nominated contacts sounds like a good option.

Cheers
